Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -178,11 +174,15 @@ open class BasicFormat: SyntaxRewriter {
     }
 
     switch (first?.tokenKind, second?.tokenKind) {
+    case (.leftBrace, .rightBrace):  // empty declarations like `struct {}`.
+      return false
```

Comment:
I would prefer this to be a whitelist that always returns `true` if something matches and only falls through to `return false`. In my experience those are a lot easier to reason about than if you interleave white and blacklists.

So, I’d prefer this to change the below to something like here

https://github.com/apple/swift-syntax/blob/3ed9dcf27c69b991d3cd8eadd64764638f3b0dec/Sources/SwiftBasicFormat/BasicFormat.swift#L260


---

When seeing the examples in the test case below, I think I changed my mind that I prefer to add a newline in between `{` and `}`. I prefer to write `func foo() {}` if I’m deliberately declaring an empty function but in the cases below, we are mostly adding `{` and `}` for Fix-Its and in that case you almost always want to add something between the braces.

Reply:
I will revert the change and update the two previously failing test cases